networkx/algorithms/shortest_paths/tests/test_weighted.py

```diff
@@ -338,6 +338,14 @@ def test_negative_edge_cycle(self):
         G.add_edge(9, 10)
         pytest.raises(ValueError, nx.bidirectional_dijkstra, G, 8, 10)
 
+    def test_negative_edge_cycle_custom_weight_key(self):
+        d = nx.DiGraph()
+        d.add_node("a")
+        d.add_node("b")
```

Comment:
These two lines aren't needed because the `add_edge` automatically adds the nodes if needed.

Reply:
Done.